### ISSUE TITLE
 loss = -nan 

### ISSUE THREAD

**Issue by ParhamKhojasteh** [BVLC/caffe#4449]
I'm trying to train my Net by my architecture and also my own data set which converted to LMDB. while training my net result is like: 
 I0712 12:10:30.398227  3123 solver.cpp:404]     Test net output #0: accuracy = 0
I0712 12:10:30.403271  3123 solver.cpp:404]     Test net output #1: loss = 2.48997 (* 1 = 2.48997 loss)
I0712 12:10:37.058244  3123 solver.cpp:228] Iteration 0, loss = 2.46876
I0712 12:10:37.058358  3123 solver.cpp:244]     Train net output #0: loss = 2.46876 (* 1 = 2.46876 loss)
I0712 12:10:37.058405  3123 sgd_solver.cpp:106] Iteration 0, lr = 0.01
I0712 12:18:40.334496  3123 solver.cpp:228] Iteration 100, loss = -nan
I0712 12:18:40.334661  3123 solver.cpp:244]     Train net output #0: loss = -nan (* 1 = -nan loss)

I do not know what is the meaning of the loss = -nan? is my net working properly or should be some errors on it?

I used just 40 images for training and 10 images for testing. In this training section I just want to be sure that whether  the system works or not. while my data set includes more that 500000 images for training.

**Comment by YouKnowGJB** [comment]
You probably want to decrease your learning rate (by at least one magnitude).

**Comment by shelhamer** [comment]
Please ask usage and modeling questions on the mailing list.

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
